### DISCUSSION TITLE
Alternative routes

### DISCUSSION THREAD

**Discussion by ZeljkoBG** [osmandapp/OsmAnd#19952] [Ideas]
It would be good if there was a possibility to choose routes. That's the standard.

**Comment by outdoor720** [comment]
There does not seem to be a routing option fastest, most direct etc....  Another nice feature would be to route using the most twisty, hilly roads.

**Reply by ZeljkoBG** [reply]
Just choosing an economical route

**Reply by ZeljkoBG** [reply]
and that is not an alternative route. The alternative means displaying more routes...

**Comment by scaidermern** [comment]
Duplicate of #11631.
